ocrd/ocrd/processor/helpers.py

```diff
@@ -96,7 +94,8 @@ def run_processor(
         output_file_grp=output_file_grp,
         instance_caching=instance_caching
     )
-
+    
+    old_cwd = getcwd()
```

Comment:
Could you dump a stacktrace of the regression here?

Reply:
See my [analysis here](https://github.com/OCR-D/core/pull/974#discussion_r1134717424).

IMO this should be fixed here first (undoing the revert but compensating in the tests). Subsequently, it could be cherry-picked (or repeated) in #974.

Reply:
@MehmedGIT you still need to compensate in the regression tests, as described.